env-hooks/1.ros_package_path.sh.em

```diff
@@ -0,0 +1,26 @@
+# generated from ros_environment/env-hooks/1.ros_package_path.sh.em
```

Comment:
The file in roslib has these lines as well, but I don't see them here. Is the omission intended?

```
# scrub old ROS bin dirs, to avoid accidentally finding the wrong executables
export PATH="`@(PYTHON_EXECUTABLE) -c \"import os; print(os.pathsep.join([x for x in \\\"$PATH\\\".split(os.pathsep) if not any([d for d in ['cturtle', 'diamondback', 'electric', 'fuerte'] if d in x])]))\"`"
```

Reply:
Those are so old distros that I removed them intentionally. None of these legacy ROS distros shares a common platform so I don't expect that they are installed on any current system using current ROS distros.